### DISCUSSION TITLE
Data provider filter to dummy form inputs

### DISCUSSION THREAD

**Discussion by SantoshSrinivas79** [Budibase/budibase#3912] [Help] [answered]
Hi Guys! I want create some dummy form elements (like radio box, free type text).
For the radio or select box, I want to use `custom` options source and give few options that i will hardcode in the builder.

Now, I want to bind the filter on the dataprovider to use these form elements.
Is this possible using the javascript option? How to go about it?

**Comment by mjashanks** [comment, marked as answer]
Yes - this is possible, no javascript required. This should help you: https://docs.budibase.com/guides/add-a-dropdown-filter-to-a-table

Basically, your component structure should be:

```
> Form
   > Options Picker
   > Data Provider with filer using Form.Options Picker
```
